biz.aQute.bndlib/src/aQute/bnd/component/DSAnnotationReader.java

```diff
@@ -1044,6 +1044,11 @@ private void doReference(Reference reference, Annotation annotation) throws Exce
 			}
 			case PARAMETER : {
 				if (!"<init>".equals(member.getName())) {
+					if (member.isPrivate() && member.isSynthetic()) {
```

Comment:
There is something else going on here. Synthetic methods should already be ignored.

https://github.com/bndtools/bnd/blob/66b0d06e756a979b3fef97dd0470dea56b9fc4f7/biz.aQute.bndlib/src/aQute/bnd/component/DSAnnotationReader.java#L1545-L1548

So I don't see why we would even get here for a synthetic method. Can you send me the test class so I can dig in further?